docs/section6/Rule6-10.md

```diff
@@ -0,0 +1,50 @@
+
+# Lighting - Rule 6-10  
+
+**Rule ID:** 6-10   
+**Rule Description:** Where retail display lighting is included in the proposed building design in accordance with Section 9.5.2.2(b), the baseline building design retail display lighting additional power shall be equal to the limits established by Section 9.5.2.2(b) or same as proposed, whichever is less.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.1 6 Baseline 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Space  
+**Function Call:**  
+
+- get_component_by_id()
+
+## Applicability Check:  
+- look at each building segment: `for building_segment in P_RMD.building_segments:`
+    - get the building segment lighting type: `building_segment_lighting_type = building_segment.lighting_building_area_type`
+    - look at each space: `for space in building_segment...spaces:`
+        - set a boolean called applicable to false: `applicable = false`
+        - set lighting_space_type equal to the space.lighting_space_type: `lighting_space_type = space.lighting_space_type`
+        - if the lighting_space_type is NULL, set the lighting_space_type equal to the building_segment_lighting_type: `if lighting_space_type == NULL: lighting_space_type = building_segment_lighting_type`
```

Comment:
Note to self, the building area types differ from space types, maybe this does not matter